shell/platform/common/cpp/accessibility_bridge.cc

```diff
@@ -0,0 +1,525 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "accessibility_bridge.h"
+
+#include <functional>
+#include <utility>
+
+#include "flutter/third_party/accessibility/ax/ax_tree_update.h"
+#include "flutter/third_party/accessibility/base/logging.h"
+
+namespace flutter {  // namespace
+
+constexpr int kHasScrollingAction =
```

Comment:
Is it possible to export this const int in embedder.h if it uses enum in the same file?
I tried
```
FLUTTER_EXPORT const int32_t kHasScrollingAction =
    kFlutterSemanticsActionScrollLeft |
    kFlutterSemanticsActionScrollRight |
    kFlutterSemanticsActionScrollUp |
    kFlutterSemanticsActionScrollDown;
```
but it kept complaining about 
```
duplicate symbol '_kHasScrollingAction' in:
    obj/flutter/shell/platform/embedder/embedder.embedder_engine.o
    obj/flutter/shell/platform/embedder/embedder.platform_view_embedder.o
```
Do i miss something?

Reply:
It can only be defined in one place.

Reply:
yes, I remove the definition in this file and put it in embedder.h only.